sys/net/gnrc/netif/gnrc_netif_raw.c

```diff
@@ -104,13 +104,23 @@ static gnrc_pktsnip_t *_recv(gnrc_netif_t *netif)
     return pkt;
 }
 
+static gnrc_pktsnip_t *_skip_pkt_head(gnrc_netif_t *netif, gnrc_pktsnip_t *pkt)
+{
+    if (gnrc_netif_netdev_legacy_api(netif)) {
+        /* we don't need the netif snip: remove it */
+        return gnrc_pktbuf_remove_snip(pkt, pkt);
+    } else {
```

Comment:
```suggestion
    }
    else {
```

Please squash right away.